playbooks/openstack/configuration.md

```diff
@@ -1017,3 +1017,81 @@ $ ansible-playbook --user openshift \
   -i inventory \
   openshift-ansible/playbooks/openstack/openshift-cluster/install.yml
 ```
+
+
+## Opening Optional Ports
+
+Certain features of Openshift that are not part of the default configuration may require ports to be opened. The following changes to openshift-ansible/roles/openshift_openstack/defaults/main.yml are needed to enable these features. This section assumes that these services were already properly enabled and configured in OSEv3.yml or all.yml, and will not include details pertaining to them.
```

Comment:
From this, it is not clear to me where are you supposed to include this extra lines. Are you supposed to add a var on all.yml containing this (and all the default ones not being modified)? Or are you suggesting adding them directly to the defaults/main.yml file? I don't think the second approach is valid. And if the first, it would be nice to have some extra inputs on the documentation to make it clearer

Reply:
I will go over it and make it clearer.